### ISSUE TITLE
Install standalone Portal

### ISSUE THREAD

**Issue by cdbarker** [Esri/arcgis-powershell-dsc#298]
Is it possible to install a standalone portal/web adaptor without a hosting server?

Our most recent attempt resulted in a failure during web adaptor configuration because it prompts for credentials for ServerPrimarySiteAdminCredential. As there is no server, this parameter hasn't been set in the json.

Is there an assumption that there will always be a hosting server?

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Module Version

* 3.0.0

### Affected Resource(s)

* ArcGIS_Server
* ArcGIS_Portal

### Configuration Files

```
# Copy-paste your DSC JSON configurations here - for large configs,
# please use a service like Dropbox and share a link to the ZIP file.
```

**Comment by cameronkroeker** [comment]
Hello @cdbarker ,

This is a bug and will be addressed in a future release. In the meantime, feel free to use the following workaround:

In file Modules/ArcGIS/Configurations-OnPrem/ArcGISWebAdaptor.ps1 remove the line `[ValidateNotNullorEmpty()]` above the  `$ServerPrimarySiteAdminCredential` param declaration (line 4).

https://github.com/Esri/arcgis-powershell-dsc/blob/50a5cf49476a7059ea417dcc67f15c76e4d41522/Modules/ArcGIS/Configurations-OnPrem/ArcGISWebAdaptor.ps1#L4

Thanks,
Cameron K.

**Comment by cameronkroeker** [comment]
Hello @cdbarker,

This issue has been addressed in the latest release, v3.2.0.

https://github.com/Esri/arcgis-powershell-dsc/releases/tag/v3.2.0

Thanks,
Cameron K.
